### PR TITLE
Hide tabs for external and event types

### PR DESCRIPTION
**Pull request by jivey**
- Don't render Submission and Scores tabs for External and Event types
- Fix an issue with External type in the modal not going to assignment review 

image

### CODE REVIEW

File: tutor/src/screens/assignment-review/ux.js [hunk 1]
```diff
@@ -254,4 +254,8 @@ export default class AssignmentReviewUX {
     return Boolean(['reading', 'homework'].includes(this.planScores.type));
   }
 
+  @computed get canDisplaySubmissionAndScoresTabs() {
+    return Boolean(['reading', 'homework'].includes(this.planScores.type));
+  }
+
```
Comment (camliu89) [inline tutor/src/screens/assignment-review/ux.js:260]: @jivey isn't this returning the same thing as the function above?

Comment (jivey) [reply tutor/src/screens/assignment-review/ux.js:260]: @camliu89 I thought about DRYing them into a single method but couldn't decide on a good name to refer to these two types. "Workable"? Let me know if you have any suggestions!

Comment (camliu89) [reply tutor/src/screens/assignment-review/ux.js:260]: mmm. I agree with you. I don't like long names. mmm. "hasReadingOrHomeworkType"? It is what is checking for :thinking:

Comment (jivey) [reply tutor/src/screens/assignment-review/ux.js:256]: That's way better and simpler! 😄

**Pull request merged**